splink/comparison_library_utils.py

```diff
@@ -60,3 +60,64 @@ def datediff_error_logger(thresholds, metrics):
         raise ValueError("\n\n".join(error_logger))
 
     return
+
+
+def distance_threshold_comparison_levels(
+    self,
+    col_name: str,
+    distance_function_name: str,
+    distance_threshold_or_thresholds,
+    higher_is_more_similar: bool = True,
+    m_probability_or_probabilities_thres: list = None,
+):
+    thresholds = ensure_is_iterable(distance_threshold_or_thresholds)
+    threshold_comparison_levels = []
+
+    if m_probability_or_probabilities_thres is None:
+        m_probability_or_probabilities_thres = [None] * len(thresholds)
+    m_probability_or_probabilities_thres = ensure_is_iterable(
+        m_probability_or_probabilities_thres
+    )
+
+    for thres, m_prob in zip(thresholds, m_probability_or_probabilities_thres):
+        if distance_function_name == "levenshtein":
+            distance_function_name = self._levenshtein_name
+            higher_is_more_similar = False
+        if distance_function_name == "jaro":
+            distance_function_name = self._jaro_name
+            higher_is_more_similar = True
+        if distance_function_name == "jaro-winkler":
+            distance_function_name = self._jaro_winkler_name
+            higher_is_more_similar = True
+        if distance_function_name == "jaccard":
+            distance_function_name = self._jaccard_name
+            higher_is_more_similar = True
```

Comment:
Should these be `elif`'s so we don't check every condition?

Reply:
Yep, that was an oversight on my part. Fixed